python/benchmarks/common.py

```diff
@@ -19,7 +19,6 @@
 import decimal
 from functools import partial
 import itertools
-import os
```

Comment:
flake8 fails on line 41 as it's Python3 only. We should either only execute it for Python 3 or make it Python2 compatible.

Reply:
fixed